pydantic/root_model.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import annotations as _annotations
+from __future__ import annotations as _annotations  # noqa: D100
```

Comment:
Why?

Reply:
Actually, it was because we didn't have the doc at the first line of the file and my IDE fixed it by ignoring it.

Fixed by adding doc for the first line.